GliaWidgets/Component/Header/HeaderStyle.swift

```diff
@@ -11,19 +11,43 @@ public struct HeaderStyle {
     /// Background color of the view.
     public var backgroundColor: UIColor
 
+    /// Back button style in the header.
```

Comment:
I should've read these more carefully before suggesting taking them directly from other styles 😅 
"in the header" is of course redundant here - it's `HeaderStyle` after all.
Let's make these consistent with `endButton` and `endScreenShareButton`:
`Style of the back button.`